src/Auth0.AuthenticationApi/Models/AssociateNewAuthenticatorResponse.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Auth0.AuthenticationApi.Models
+{
+    public class AssociateNewAuthenticatorResponse
+    {
+        [JsonProperty("oob_code")]
+        public string OobCode { get; set; }
+
+        [JsonProperty("binding_method")]
+        public string BindingMethod { get; set; }
+
+        [JsonProperty("secret")]
+        public string Secret { get; set; }
+
+        [JsonProperty("barcode_uri")]
+        public string BarcodeUri { get; set; }
+
+        [JsonProperty("authenticator_type")]
+        public string AuthenticatorType { get; set; }
+
+        [JsonProperty("oob_channel")]
+        public string OobChannel { get; set; }
```

Comment:
This is not as mentioned in the docs, as the docs mention `oob_channels` for the response. I think the code is correct, but the docs is not. I will need to verify this with the corresponding team.